src/components/fonts.css

```
@@ -0,0 +1,62 @@
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Golos Text';
+  src: url('../assets/fonts/Golos/GolosText-Black.ttf') format('truetype');
+  font-weight: 900;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Inter';
```

Stylelint (17.12.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)